spec/system/profiles_spec.rb

```diff
@@ -17,5 +17,36 @@
       expect(page).to have_content '私の名前は空条承太郎です'
       expect(page).to have_content 'google.com'
     end
+
+    it '自分のプロフィールを編集できること' do
```

Comment:
（細かい）
context で自分のプロフィールとあるので、省略してもよさそう
```suggestion
    it '編集できること' do
```